entwatch/ze_ffxii_feywood_b6_2k_override.cfg

```diff
@@ -224,8 +224,8 @@
 		"blockpickup"		"false"
 		"allowtransfer"		"false"
 		"forcedrop"		"false"
-		"chat"			"true"
-		"hud"			"true"
+		"chat"			"false"
+		"hud"			"false"
```

Comment:
Reasoning?

Reply:
The reason was because it only gives the player who picks up the item gives them speed, its the same type of reason as westersand v7, i forgot i accidently added a func_button when there is no func_button.